src/game/combinator.rs

```diff
@@ -0,0 +1,197 @@
+#![allow(dead_code)]

```

Comment:
i should arguably remove the ones from the functions and only have the global so that at the right time we can remove the global and clean up the dead code...

Reply:
i just pushed a change to pull all redundant local [allow*]